example/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <=3.0.0"
```

Comment:
Support current dart version `3.0.5`?

Reply:
maybe 
```
sdk: ">=2.12.0 <4.0.0"
```

Reply:
i have update the range sdk version, kindly check